pkg/detectors/godaddy/v1/godaddy.go

```diff
@@ -0,0 +1,145 @@
+package godaddy
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+
+	regexp "github.com/wasilibs/go-re2"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct {
+	client *http.Client
+}
+
+type GoDaddyEnv string
```

Comment:
nit: Since the package is already named `godaddy`, we could rename this to `env` to avoid stuttering. It also seems safe to unexport it.

[ref](https://google.github.io/styleguide/go/decisions#package-vs-exported-symbol-name)